CHANGES/8507.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed blocking IO in the event loop for static resources and refactored exception handling -- by :user:`steverep`.
```

Comment:
Could you update this with more context? When you say “Fixed blocking I/O”, this implies that it was broken somehow but avoids specifics. I think that the end-users reading the release notes deserve some background regarding how this affects them on the high level, without having to dig into the source code.
Would you be able to compose something for that audience?

Reply:
I added context and changed "fixed" to "removed".

Reply:
Thanks! Marking this as resolved.